### PR TITLE
:building_construction: exampleSite: Document key features (closes #119)

### LINKED ISSUE #119: As a developer, I want key features to be clearly documented so that I can effectively use available features.
### Summary

Related to #89.

### Priority

secondary

### Category

documentation

### Type

business

### PR DESCRIPTION
**Pull request by jwflory**
This change is a large reworking of content, structure, and organization. Several unique features of the UNICEF Inventory theme are given a feature profile to show how they work and how to use them. This makes features easier to discover and also makes them searchable from the site homepage.

Additionally, a "lorem ipsum" page is included in this change. It groups several page elements together in a single page, to help a webmaster evaluate the theme's appearance and for developers to test for regressions when introducing new front-end changes.

Blocked by #148. Closes #119.

### CODE REVIEW

File: exampleSite/config.yaml [hunk 1]
```diff
@@ -107,17 +107,16 @@ params:
...
   footer:
-    mainSite: https://www.unicefinnovationfund.org/
-    mainSiteName: UNICEF Innovation Fund
-    description: "This is metadata for the site footer. Set in Hugo config file under \"brand and identity information\"."
+    mainSite: https://www.unicef.org/innovation/
+    mainSiteName: UNICEF Office of Innovation
 
 
 # --- social platform settings ---
 # themify icon pack : https://themify.me/themify-icons
```
Comment (Neha9849) [inline exampleSite/config.yaml:116]: I think the line breaks here are not uniform with rest of the code

File: exampleSite/config.yaml [hunk 2]
```diff
@@ -171,51 +172,54 @@ Languages:
...
-          weight: 20
+          weight: 10
           url: faq
-        - name: Contacter
-          weight: 30
+        - name: Contactar
+          weight: 20
           url: contact
 
         - name: pages
```
Comment (Neha9849) [inline exampleSite/config.yaml:225]: Same goes here ( line breaks)

**Review by Neha9849 — COMMENTED** [review]
Great work🎉.This is really cool documentation✨

File: exampleSite/config.yaml (hunk 1 above)
Comment (jwflory) [reply exampleSite/config.yaml]: Addressed in commit bea853a3c63fbd76a36803126fd5336fd5cdd9ad. The convention used is two lines preceding a header comment (i.e. `# --- header ---`) and one line after a header begins, in order to denote the section. The config is now consistent in using that convention.

File: exampleSite/config.yaml (hunk 2 above)
Comment (jwflory) [reply exampleSite/config.yaml]: Addressed in commit bea853a3c63fbd76a36803126fd5336fd5cdd9ad.

**Comment by jwflory** [comment]
Thanks for the review @Neha9849! 🚀 I addressed your feedback in a new commit. I'm going to squash down these commits and merge. 🌊

**Pull request merged**